ledger/accountdb.go

```diff
@@ -825,10 +695,15 @@ func removeEmptyAccountData(tx *sql.Tx, queryAddresses bool) (num int64, address
 	return num, addresses, err
 }
 
-func createAccountExtTable(tx *sql.Tx) error {
-	tableCreate := `CREATE TABLE IF NOT EXISTS accountext (
-		id integer primary key,
-		data blob)`
+func updateToExtendedAccountGadgetTable(tx *sql.Tx) error {
+	tableCreate := `CREATE TABLE IF NOT EXISTS accountcreatabledata (
+		addrid integer NOT NULL,
+		creatable integer NOT NULL,
+		ctype integer NOT NULL,
```

Comment:
Performance will be better if we had multiple tables instead of a generic one containing `ctype`.

Reply:
proof?

Reply:
No rowid tables store the whole row data in the intermediary b-tree nodes. By increasing the size of the data, b-tree nodes will store less nodes, so the height of the tree will be bigger.